chapter5-tensorrt-api-basics/5.3-infer-model/src/cpp/utils.hpp

```diff
@@ -64,7 +64,6 @@ static void __log_info(Level level, const char* format, ...) {
     va_end(args);
 
     if (level == Level::ERROR) {
-        free(msg);
         exit(1);
```

Comment:
Great! Thank you for helping me to find this unnecessary memory free for stack. Btw, I realized that may be it is not good to exit the program from the logger. We should create an `assert` to terminated the program with unexpected behavior. So my suggestion is that we also need to remove the `exit(1)` here.